docs/proposals-accepted/20240909-live-blocks-in-objectstorage.md

```diff
@@ -0,0 +1,62 @@
+---
+type: proposal
+title: Avoid querying blocks in object storage if they have a live source
+status: approved
+owner: MichaHoffmann
+menu: proposals-accepted
+---
+
+## Avoid querying blocks in object storage if they have a live source
+
+> TL;DR: We propose to not query blocks in object storage that are still served by a Sidecar/Ruler or Receiver.
```

Comment:
Can you motivation/cases for why we don't want to fan out to the StoreGW?
One con I see for this, is that some of our write path components being under more read pressure

Reply:
Ill add a note to the document but this should not increase read pressure on any source component since you fan-out to them anyway ( though we need to respect "--min-time" and "--max-time" configurations, thats a good point.